adidas-ascii-logo/swift/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
+
+import adidas_ascii_logoTests
+
+var tests = [XCTestCaseEntry]()
+tests += adidas_ascii_logoTests.allTests()
+XCTMain(tests)
```

Comment:
It is recommended to add a blank line at the end of the file.

Reply:
The `.editorconfig` is already set to do it.

Reply:
Done